jsonb/2.0/apidocs/allclasses.html

```diff
@@ -2,10 +2,9 @@
 <!-- NewPage -->
 <html lang="en">
 <head>
-<!-- Generated by javadoc (11.0.7) on Thu Jun 04 13:06:03 GMT 2020 -->
-<title>All Classes (JSON-B API 2.0.0-RC2 API)</title>
+<!-- Generated by javadoc -->
+<title>All Classes (JSON Processing API documentation)</title>
```

Comment:
This is what my earlier comment referred to...  All of the jsonb apidocs seem to have been replaced with jsonp apidocs.  Oops.